src/plugins/intel_gpu/src/graph/impls/onednn/convolution_onednn.cpp

```diff
@@ -63,12 +65,38 @@ struct convolution_onednn : typed_primitive_onednn_impl<convolution, dnnl::convo
             auto a_zp = instance.activations_zero_points_memory(0);
             dnnl::memory::desc desc = onednn::layout_to_memory_desc(a_zp->get_layout(), dnnl::memory::format_tag::a, true);
             args.insert({DNNL_ARG_ATTR_ZERO_POINTS | DNNL_ARG_SRC, a_zp->get_onednn_memory(desc)});
+
+            GPU_DEBUG_GET_INSTANCE(debug_config);
+            GPU_DEBUG_IF(debug_config->verbose >= 2) {
+                size_t desc_size = desc.get_size();
+                std::vector<int32_t> debug_buf(desc_size);
+                auto dnnl_mem = a_zp->get_onednn_memory(desc);
+                void *mapped_ptr = dnnl_mem.map_data();
+                if (mapped_ptr) {
+                    std::memcpy(debug_buf.data(), mapped_ptr, desc_size);
+                    GPU_DEBUG_COUT << instance.get_node().id() << " activations_zero_points: " << debug_buf[0] << std::endl;
```

Comment:
what about if it is not scalar but per-channel?

Reply:
Updated to print all values from desc size.